contenus/actualites/2021-12-22-vaccination-enfants.toml

```
@@ -0,0 +1,4 @@
+date = 2021-12-22
+texte = "la vaccination est désormais possible pour les enfants de 5 à 11 ans"
```

```suggestion
texte = "la vaccination est désormais possible pour les enfants à partir de 5 ans"
```

J'aurais envie d'appuyer sur la borne inférieure sans pour autant borner au-dessus 🤔 (juste pour l'actualité)